docs/content/preview/yugabyte-voyager/known-issues/_index.md

```diff
@@ -53,3 +53,96 @@ Currently, yb-voyager doesn't support the following features:
+
+- `JSON_TABLE` ([JSON Query function](../known-issues/postgresql/#postgresql-12-and-later-features)) usage in DML statements.
+- [Non-decimal integer literals in DML statements.](../known-issues/postgresql/#postgresql-12-and-later-features)
+- [Two-Phase Commit (XA syntax)](../known-issues/postgresql/#two-phase-commit).
+
+For example:
+
+```output.sql
+--query stored in pg_stat_statements:
+SELECT $1 as binary; 
+
+--actual query:
+SELECT 0b101010 as binary;
+```
+
+### Transactional queries in pg_stat_statements
+
+pg_stat_statements records a single transaction as multiple separate query entries. This fragmentation makes it challenging to detect issues that occur within transaction boundaries, such as:
+
+- [DDL operations within Transaction](../known-issues/postgresql/#ddl-operations-within-the-transaction)
+
+**Example:**
+
+```output.sql
+yugabyte=# \d test
+Did not find any relation named "test".
+yugabyte=# BEGIN;
+BEGIN
+yugabyte=*# CREATE TABLE test(id int, val text);
+CREATE TABLE
+yugabyte=*# \d test
+                Table "public.test"
+Column |  Type   | Collation | Nullable | Default 
+--------+---------+-----------+----------+---------
+id     | integer |           |          | 
+val    | text    |           |          | 
+yugabyte=*# ROLLBACK;
+ROLLBACK
+yugabyte=# \d test
+                Table "public.test"
+Column |  Type   | Collation | Nullable | Default 
+--------+---------+-----------+----------+---------
+id     | integer |           |          | 
+val    | text    |           |          | 
+```
+
+### Determining the type during query processing
+
+In complex queries, determining the type of data being handled is not always straightforward. This limitation affects the detection of [JSONB subscripting](../known-issues/postgresql/#jsonb-subscripting) in such queries, making it difficult to report issues accurately.
+
+For example:
+
+```output.sql
+select ab_data['name'] from (select data as ab_data from test_jsonb);
+# data is the JSONB column in test_jsonb table
+```
+
+### Parser limitations
+
+The internal [Golang PostgreSQL parser](https://github.com/pganalyze/pg_query_go) used for schema analysis and migration assessment has certain limitations within PL/pgSQL blocks. This leads to not being able to detect issues on the statements embedded inside Expressions, Conditions, Assignments, Loop variables, Function call arguments, and so on.
+
+For example:
+
+```output.sql
+CREATE OR REPLACE FUNCTION example() 
+RETURNS VOID AS $$
+DECLARE
+    x TEXT; -- Adjust the type based on jsonb value type
+BEGIN
+    x := (SELECT jsonb_column['value'] FROM orders WHERE id = 1);
+    -- Further processing logic here
+    RAISE NOTICE 'Extracted value: %', x;
+END;
+$$ LANGUAGE plpgsql;
+```
+
+### Global objects
+
+Global objects in PostgreSQL are database-level objects that are not tied to a specific schema. Some of the following objects are not fully supported in schema analysis and migration assessments for detecting issues.
```

Comment:
"Some of the following objects" - why not list only the ones not supported?
This sentence is confusing generally, can you rephrase, preferably in active voice, I don't understand what isn't supported exactly
@priyanshi-yb

Reply:
Okay, maybe it can be rephrased like this 
`Global objects in PostgreSQL are database-level objects that are not tied to a specific schema. The following global objects are not supported in schema analysis and migration assessments for detecting issues.`